package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "akeneo-ts-api",
+  "version": "1.0.0",
```

Comment:
I'd probably tag this as a pre-release e.g. `0.1.0` until we nail down the API 100%.